### PR TITLE
Reference for checkoutAgreement GraphQl query

### PR DESCRIPTION
**Pull request by pmclain**
## Purpose of this pull request

Reference for checkoutAgreement GraphQl query

Fixes magento/graphql-ce#767

## Links to Magento source code

https://github.com/magento/graphql-ce/blob/c3040e682d56964b78c6e7a29388bea6c1a34c85/app/code/Magento/CheckoutAgreementsGraphQl/etc/schema.graphqls#L4-L21

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/reference/checkout-aggreements.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+Attribute | Data type | Description
+--- | --- | ---
+`agreement_id` | Integer | Checkout Agreement identifier
+`name` | String | Checkout Agreement name
+`content` | String | Checkout Agreement plaintext or HTML content
+`content_height` | String | CSS height of Checkout Agreement
+`checkbox_text` | String | Checkout Agreement checkbox label
+`is_html` | Boolean | Is Checkout Agreement content in HTML format
+`mode` | [CheckoutAgreementMode](#checkoutAgreementMode) | An array of regions within a particular country
```
Comment (keharper) [inline guides/v2.3/graphql/reference/checkout-aggreements.md]:
```suggestion
`mode` | [CheckoutAgreementMode](#checkoutAgreementMode) | Indicates whether terms and conditions are applied manually (`MANUAL`) or automatically (`AUTO`)
```

File: guides/v2.3/graphql/reference/checkout-aggreements.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+### CheckoutAgreementMode enumerable {#checkoutAgreementMode}
```
Comment (keharper) [inline guides/v2.3/graphql/reference/checkout-aggreements.md]: Delete this section

File: guides/v2.3/graphql/reference/checkout-aggreements.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+---
+group: graphql
+title: Directory endpoint
+contributor_name: Something Digital
+contributor_link: https://www.somethingdigital.com/
+---
+
+To retrieve enabled checkout agreements. The query will always return an empty array when
+`checkout/options/enable_agreements` is disabled.
+
```
Comment (keharper) [inline guides/v2.3/graphql/reference/checkout-aggreements.md]: Create a head2 labeled Output attributes with the following sentence:

The `CheckoutAgreements` object contains an array of `CheckoutAgreement` objects.

(or something else, if you can think of anything better to say).

Move the Output attributes section to the end.

Comment (pmclain) [reply guides/v2.3/graphql/reference/checkout-aggreements.md]: Can you take a look and make sure this is the intended result? https://github.com/magento/devdocs/pull/4943/commits/dbd8ee2ced219112516709d2564a31238e4b2051#diff-b903ec86dd935702a7085d632c676d8dR69

**Comment by keharper** [comment]
@pmclain For 2.3.3, I'm getting rid of all "XYZ endpoint" topics. Instead, I'll document each query and mutation separately.  I've barely begun this process, and I haven't created any query docs, but you can see the first of the mutations here:

https://github.com/magento/devdocs/blob/develop/guides/v2.3/graphql/mutations/apply-store-credit.md

I'd like each topic to have this layout:

* Introductory paragraphs (no heading)
* Example usage
* Input attributes (iff applicable)
* Output attributes

If you would move the *CheckoutAgreement attributes* section to the bottom, I'll be ready to merge this PR. (I'll move the topic and add navigation later.)

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
Thanks @pmclain

**Pull request merged**